### ISSUE TITLE
segmentation fault

### ISSUE THREAD

**Issue by Kayv-cmb** [cortex-lab/phy#1231]
When I try to open the gui-template with `phy template-gui ./params.py` nothing happen, I have an error message saying segmentation fault.
Could I have some help on how to resolve that ? I tried different dataset, I tried on a cluster with 250GB of RAM, I tried locally on my mac with 16 GB of ram, I have the access right to the file. I am kinda lost on what to do?

full error message :
```
phy template-gui ./params.py
15:51:40.373 [W] model:603            Unreferenced clusters found in templates (generally not a problem)
15:51:42.821 [W] model:625            Unreferenced clusters found in spike_clusters (generally not a problem)
zsh: segmentation fault  phy template-gui 
```

**Comment by zm711** [comment]
@Kayv-cmb.

Couple questions.

How did you install phy? Could you do 
```python
conda list
```
inside your environment if you installed with anaconda. 

then could you run

```bash
phy template-gui ./params.py --debug
```

and copy the output so we can get more resolution of what exactly is calling the seg fault.

**Comment by Kayv-cmb** [comment]
Thanks for answers,

Before I had installed it with the `pip install git+https://github.com/cortex-lab/phy.git`

I managed to solve the SegFault by installing through the .yml file. Now I have this issue
```
QOpenGLFramebufferObject: Unsupported framebuffer format.
QOpenGLFramebufferObject: Framebuffer incomplete, missing attachment.
```
I think it's because I run it on a cluster. I managed to run it locally so everything is fine. 

Also I think phy is not compatible with python 3.12 as they removed imp module
